index.html

```diff
@@ -48,6 +48,164 @@ <h2 class="title">I'm a Software Developer</h2>
             <img class="imgg" src="img/v13.png">
             <img class="imgh" src="img/v13.png">
         </div>
+    </section><section class="work">
+        <div class="work-title1">
+            <h1 class="work-head">My Recent Works</h1>
+            <img class="work1-img" src="img/Img1.png">
+            <h2 class="work-head2">Multi-Post Stories</h2>
+            <p class="work1-para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <div class="work-menu">
+                <ul class="work1-list">
+                    <li class="p">css</li>
+                    <li>html</li>
+                    <li>bookstrap</li>
+                    <li>Ruby</li>
+                </ul>
+            </div>
+            <a href="#" class="one btn">See Project</a>
+        </div>
+        <div class="work-title2">
+            <h1 class="work-head22">Professional Art Printing Data</h1>
+            <p class="work2-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="work2-menu">
+                <ul class="work2-list">
+                    <li class="ph">html</li>
+                    <li>bookstrap</li>
+                    <li>Ruby</li>
```

Comment:
Well done :+1:  for implementing some of the changes requested by the previous reviewer. However, there is a blank white empty space to the right side of the page as indicated below :arrow_down:  which shows that the width of your page is fixed and still not responsive for the mobile version. 
Kindly consider updating the width of your mobile page to fit the whole width to remove the blank space :pray: 


![Screenshot from 2022-12-01 14-57-40](https://user-images.githubusercontent.com/65233860/205087651-dcacf591-e24d-4575-a1b7-7e80e9d475a9.png)

![Screenshot from 2022-12-01 15-01-25](https://user-images.githubusercontent.com/65233860/205087685-fdd8fcb0-b6fe-4cc8-bd6e-14747d956699.png)